docs/adrs/ADR-1.md

```diff
@@ -13,29 +12,97 @@ Deepansh
+1. Where should the homework sessions data be stored on BQ? Should it be stored in the `test_responses` table that exists or some new table?
+2. If storing in a different table, how should we name the new table? The old one is `test_responses`. It doesn't have the keyword `assessments` in the name.
+3. Is the question level analysis also needed for homework tests? If yes, then should it be stored in a separate table or in the table for `assessments`?
+4. This data also needs to go into the Dynamo DB (for reporting engine). Should it go to a different table or the same existing one?
+5. What all fields are necessary or good to have for the homework level sessions? The fields listed below are the ones present in the assessments table. We can filter out or add more fields for the homework quizzes. Decision needs to be taken on entries with a :question: emoji:
+
+    | Field | Comments |
+    | ------ | ----------- |
+    | user_id | will be added |
+    | num_skipped | :question: Right now one can't skip in homework mode. But it's possible in the future, so for now, should we keep it and the value will be 0? |
+    | num_correct | :question: Number of correctly answered graded questions. This will not include ungraded questions right? |
+    | num_wrong | :question: Number of wrongly answered graded questions. This will not include ungraded questions right? |
+    | marks_scored | :question: Marks scored by the user on graded questions wrt marking scheme. What is the default marking scheme if nothing is present? |
+    | percentage | Marks scored by user / total marks possible |
+    | section | :question: Will these questions also have subject wise sections like P/C/M/B? |
+    | max_marks_possible | total graded questions * score for correct answer |
+    | user_data_validated | does the user id exist in auth layer logs |
+    | avg_test_score | avg of scores of users in a particular session |
+    | highest_test_score | highest of scores of users in a particular session |
+    | rank | rank of student among other students in a particular session |
+    | percentile | percentile of student among other students in a particular session |
+    | num_test_takers | number of unique users who attempted the quiz |
+    | session_id | unique session id |
+    | start_date | start date of session |
+    | start_time | start time of session |
+    | end_date | end date of session |
+    | end_time | end time of session |
+    | infinite_session | :question: Should we add this? if yes, then this needs to go into firestore sessions DB as well |
```

Comment:
Yes, why not. Unless we're 100% sure infinite sessions will not exist for homework

Reply:
Will talk to Karn and Heena